book/src/performance-metrics.md

```diff
@@ -0,0 +1,25 @@
+# Performance Metrics
+
+Solana cluster performance is measured as average number of transactions per second
+that the network can sustain (TPS). And, how long it takes for a transcation to be
+confirmed by super majoring of the cluster (Confirmation Time).
+
+Each cluster node maintains various counters that are incremented on certain events.
+These counters are periodically uploaded to a cloud based database. Solana's metrics
+dashboard fetches these counters, and computes the performance metrics and displays
+it on the dashboard. 
+
+## TPS
+
+The leader node's bank maintains a count of transactions that it processed. The dashboard
+displays sum of the count for every 2 second period in the TPS time series graph.
+The dashboard also shows mean, maximum and total TPS as a running counter.
+
+## Confimation Time
+
+Each validator node maintains a list of active ledger forks that are visible
+to the node. Whenever a fork's final tick is registered, the fork is frozen, and
```

Comment:
That’s not correct.  The validator confirmation time for a fork is based on the difference when a fork is created and when the first child fork that has a cumulative supermajority vote for the original fork s frozen.

Reply:
I understand that fork is frozen when there's a cumulative supermajority vote. I don't understand why the child fork has to be frozen?

Reply:
Frozen right now means that blocktree received all the blobs and the bank processed them.  Validator confirmation time for a parent is based on when any descendant of the parent is frozen and that branch has a supermajority conforming the parent.  Replay stage doesn’t compute a rolling vote total on incomplete forks.

Reply:
ok. that makes sense. I had interpreted your last comment as one of the child fork has to receive supermajority vote. Just to confirm, this is when a fork is considered confirmed:

1. The fork itself has received cumulative supermajority vote.
2. One of the child of the fork has been frozen (i.e. received all it's blocks)